### ISSUE TITLE
View scaling on separate (esp large) devices

### ISSUE THREAD

**Issue by sarajw** [responsively-org/responsively-app#683]
# 🚀 Feature Request

### 📝 Description

I would like to see how a page shows up in full screen on huge devices, like 4k or 5k, but that being larger than the monitor I have, it makes for an uncomfortable scroll-around situation.

### ✨ Describe the solution you'd like

If the devices could be scaled separately (not just zooming out on all of them at once), that would be wonderful.

### ✍️ Describe alternatives you've considered

Only to zoom out on everything at once. I'm not sure there are any other workarounds?

**Comment by jjavierdguezas** [comment]
have you tried Individual Layout + Zoom ?
![image](https://user-images.githubusercontent.com/13673443/171008100-978f850d-4260-4629-b7a4-bf0a36caf92b.png)

**Comment by sarajw** [comment]
Yes true, that works. It could just be cool to see an emulated large screen all together with the other ones - that's why I posted it as a possible feature, not a bug :)
